transport_nantes/authentication/templates/registration/password_reset_form.html

```diff
@@ -0,0 +1,45 @@
+{% extends 'asso_tn/base_transport_nantes.html' %}
+{% load static %}
+
+
+{% block app_content %}
+
+<style type="text/css">
+  .form-signin {
+    width: 100%;
+    max-width: 330px;
+    padding: 15px;
+    margin: auto;
+  }
+  .form-signin .checkbox {
+    font-weight: 400;
+  }
+  .form-signin .form-control {
+    position: relative;
+    box-sizing: border-box;
+    height: auto;
+    padding: 10px;
+    font-size: 16px;
+  }
+  .form-signin .form-control:focus {
+    z-index: 2;
+  }
+  .form-signin input[type="email"] {
+    margin-bottom: 10px;
+    border-bottom-right-radius: 0;
+    border-bottom-left-radius: 0;
+  }
+
+  .h3{
```

Comment:
Why is this a style on all h3's rather than just centering some specific h3?

Reply:
Actually, I used a ready to use template and I forgot to remove CSS